CRM/Custom/Import/Form/MapField.php

```diff
@@ -6,7 +6,7 @@
 class CRM_Custom_Import_Form_MapField extends CRM_Contact_Import_Form_MapField {
   protected $_parser = 'CRM_Custom_Import_Parser_Api';
   protected $_mappingType = 'Import Multi value custom data';
-  protected $_highlightedFields = array();
+//  protected $_highlightedFields = array();
```

Comment:
I just looked how it was done in Contact import and tried to stay in line with that code.